src/Module/Conversation/Community.php

```diff
@@ -113,6 +113,21 @@ public static function content(array $parameters = [])
 
 		$o .= $pager->renderMinimal(count($items));
 
+		DI::page()['aside'] .= Renderer::replaceMacros(Renderer::getMarkupTemplate('widget/community_accounts.tpl'), [
+			'$content'               => self::$content,
+			'$title'                 => DI::l10n()->t('Accounts'),
+			'$all'                   => DI::l10n()->t('All Accounts'),
+			'$all_selected'          => ($parameters['accounttype'] == '') ? 'selected' : '',
```

Comment:
I'd prefer these conditions to be in the template. Just pass `$parameters['accounttype']` to the template and sort the `selected` HTML attribute display in the HTML template instead of in the PHP code.